### ISSUE TITLE
copy expert should support sql arguments

### ISSUE THREAD

**Issue by jasen-b** [psycopg/psycopg2#529]
I would like to copy from a query with output in CSV format and with a variable specifing paramtters to the query.

 something like:

    COPY ( SELECT a,b,c FROM %t JOIN myothertab ON f = g WHERE h=%s ORDER BY c,a,b ) TO STDOUT WITH CSV HEADER

  so, please add an args parameter to copy_expert

**Comment by dvarrazzo** [comment]
You can use the [psycopg2.sql](http://initd.org/psycopg/docs/sql.html) module available in psycopg 2.7

```python
from psycopg2 import sql
cur.copy_expert(
    sql.SQL("""
        COPY (
            SELECT a,b,c FROM {t} JOIN myothertab ON f = g WHERE h={h} ORDER BY c,a,b
        ) TO STDOUT WITH CSV HEADER
    """).format(
        t=sql.Identifier("My Table"),
        h=sql.Literal("foo'bar")),
    sys.stdout)
```

This will generate:
```
COPY (
    SELECT a,b,c FROM "My Table" JOIN myothertab ON f = g WHERE h='foo''bar' ORDER BY c,a,b
) TO STDOUT WITH CSV HEADER
```

(I think documentation that `sql` objects work with `copy_expert()` too is missing... adding it).

**Comment by roytalari** [comment]
I think this is closed
